src/translator/quality_estimator.cpp

```diff
@@ -0,0 +1,133 @@
+#include <sstream>
+#include "quality_estimator.h"
+
+
+
+namespace marian {
+namespace bergamot {
+
+QualityEstimator::QualityEstimator(const char* file_parameters) {
```

Comment:
Provide two means of construction:

1. `Service` supplies a string filename, from which the contents are loaded.
2. `Service` supplies an `AlignedMemory`, which is equivalent to the file-contents and the ownership of which will be absolved by `Service` through `std::move` to `QualityEstimator`.

This constructor looks like it can be repurposed for 2, but you're scanning for null character C-style then converting it to a C++ string. We can make this more compact and better.

Reply:
I implemented only the second one @jerinphilip 064cfe3 

The reason for this is that I thought that if I apply the first option, the code would be almost the same as the one through `getMemoryBundleFromConfig`